index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Steps Js Example</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/steps.css" />
+    <link rel="stylesheet" href="css/steps-mobile.css" />
+  </head>
+  <body>
+    <div class="container steps"></div>
+    <div class="container">
+      <button class="btn" onclick="steps.changeStep(true)">Previous Step</button>
+      <button class="btn" onclick="steps.changeStep()">Next Step</button>
+      <button class="btn" onclick="steps.changeStep(true, true, true)">Previous Step(valid)</button>
```

Comment:
steps.changeStep(true, true, true) it's kinda confusing

Reply:
I changed boolean parameters to one-object parameter.